### PR TITLE
Add contracts for account and Enepti Token

### PR DESCRIPTION
**Pull request by cicioflaviu**

### CODE REVIEW

File: enepti-contracts/EneptiToken.sol [hunk 1]
```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/presets/ERC721PresetMinterPauserAutoId.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract EneptiToken is ERC721PresetMinterPauserAutoId, Ownable {
+
+    constructor() ERC721PresetMinterPauserAutoId("Enepti Token", "ENEPTI", "https://enepti.com/nft/") {
+        // Starting with 10 000 ENEPTI NFTs
+        for (uint i = 0; i < 10**4; i++) {
```
Comment (Amirjab21) [inline enepti-contracts/EneptiToken.sol:10]: How much gas will this use? Is it better to allow minting by users and let the total number be limited by a constant?

File: enepti-contracts/AccountRegistry.sol [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    function updateDna(address accountAddress, uint256[] memory _dna) external hasActivatedAccount onlyAccountManager {
+         accounts[accountAddress].dna = _dna;
+    }
+
+    function getDna(address accountAddress) external view hasAccount returns(uint256[] memory) {
+         return accounts[accountAddress].dna;
+    }
+
+    modifier hasEneptiToken(uint256 nftId) {
```
Comment (Amirjab21) [inline enepti-contracts/AccountRegistry.sol:56]: does the nftId variable need to be used?

File: enepti-contracts/AccountRegistry.sol [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        require(tokenContract.isApprovedForAll(_msgSender(), address(this)), "AccountRegistry: missing approval for the NFT transfer");
+        _;
+    }
+
+    modifier isNotActive() {
+        require(!accounts[_msgSender()].active, "AccountRegistry: the account is already active");
+        _;
+    }
+
+     modifier onlyAccountManager() {
```
Comment (Amirjab21) [inline enepti-contracts/AccountRegistry.sol:81]: who is the account manager? It doesn't look like it will be set by anything

Comment (cicioflaviu) [reply enepti-contracts/AccountRegistry.sol:81]: The **Account Manager Contract** will be another layer of abstraction between the application and the account contracts

File: enepti-contracts/AccountRegistry.sol (hunk 2 above)
Comment (cicioflaviu) [reply enepti-contracts/AccountRegistry.sol:56]: Nope, I'll remove it

File: enepti-contracts/EneptiToken.sol (hunk 1 above)
Comment (cicioflaviu) [reply enepti-contracts/EneptiToken.sol:10]: I think we'll be using the Polygon Network through Moralis, which will substantially reduce the fees

**Pull request merged**